### DISCUSSION TITLE
Pylance doesn't suggest auto import for functions created with partial

### DISCUSSION THREAD

**Discussion by viliusgudziunas** [microsoft/pylance-release#5289] [Enhancement]
## Environment data

-   Language Server version: 2023.11.10
-   OS and version: Mac OS Sonoma 14.1
-   Python version (pyenv): 3.12.0
-   IDE: VS Code

## Code Snippet

```python
# package.py

from functools import partial

def my_function(arg1, arg2, arg3):
    pass

my_partial_function = partial(my_function, arg1=1, arg2=2)
```

```python
# main.py

my_partial_function(arg3=3)
```

## Repro Steps

1. Try to use auto import for `my_partial_function`

## Expected behavior

Pylance suggests a quick fix - Add "from package.py import my_partial_function"

## Actual behavior

VS Code displays "No code actions available"

**Comment by debonte** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
